### PR TITLE
Throw an error if date or number in unexpected formats

### PR DESCRIPTION
**Pull request by Flimm**

### CODE REVIEW

File: ynab_to_ledger.rb [hunk 1]
```diff
@@ -11,13 +11,27 @@ def main
...
   inflow = blank_if_zero(row["Inflow"])
   outflow = blank_if_zero(row["Outflow"])
 
   return if inflow == "" && outflow == ""
 
   month, day, year = row["Date"].split("/")
-
+  
+  begin
+    Date.new(Integer(year, 10), Integer(month, 10), Integer(day, 10))
```
Comment (pgr0ss) [inline ynab_to_ledger.rb:30]: If you're parsing as a Date, do you still need to first parse as Integer?

Comment (Flimm) [reply ynab_to_ledger.rb:30]: Yes, to avoid this error: `invalid day (not numeric)`

**Comment by pgr0ss** [comment]
Thanks!

**Comment by pgr0ss** [comment]
I added tests and then unfortunately had to revert this due to false positives: 873a412b575990c76244e85b96d0e73a10cb305d

**Pull request merged**